src/components/Invoice.tsx

```diff
@@ -0,0 +1,115 @@
+            <Typography variant="h4" component="p">
+              Recipient
+            </Typography>
+
+            <Box mt={6}>
+              <TextField id="recipient-company-name" label="Company name" variant="standard" fullWidth />
+              <TextField id="recipient-city" label="City" variant="standard" fullWidth />
+              <TextField id="recipient-street" label="Street" variant="standard" fullWidth />
+              <TextField id="recipient-postcode" label="Postcode" variant="standard" fullWidth />
+              <TextField id="recipient-nip" label="NIP" variant="standard" fullWidth />
+              <TextField id="recipient-tel" label="Tel" variant="standard" fullWidth />
+              <TextField id="recipient-email" label="E-mail" variant="standard" fullWidth />
+              <TextField id="recipient-bank-account" label="Bank Account" variant="standard" fullWidth />
+            </Box>
+          </Grid>
+          <Grid xs={6} item>
+            <Typography variant="h4" component="p">
+              Sender
+            </Typography>
+
+            <Box mt={6}>
+              {/* Pytanie Dlaczego czasem daje kursor na koniec */}
+              <TextField id="sender-company-name" label="Company name" variant="standard" fullWidth />
+              <TextField id="sender-city" label="City" variant="standard" fullWidth />
+              <TextField id="sender-street" label="Street" variant="standard" fullWidth />
+              <TextField id="sender-postcode" label="Postcode" variant="standard" fullWidth />
+              <TextField id="sender-nip" label="NIP" variant="standard" fullWidth />
+              <TextField id="sender-tel" label="Tel" variant="standard" fullWidth />
+              <TextField id="sender-email" label="E-mail" variant="standard" fullWidth />
+              <TextField id="sender-bank-account" label="Bank Account" variant="standard" fullWidth />
+            </Box>
+          </Grid>
+        </Grid>
+
+        <Grid item container>
+          <Grid item container spacing={3}>
+            <Grid item xs={6}>
+              <TextField id="name" label="Name" variant="standard" fullWidth />
+            </Grid>
+
+            <Grid item xs={1.2}>
+              <TextField type="number" id="amount" label="Amount" variant="standard" fullWidth />
+            </Grid>
+
+            <Grid item xs={1.2}>
+              <TextField id="unit" label="Unit" variant="standard" fullWidth />
+            </Grid>
+
+            <Grid item xs={1.2}>
+              <TextField type="number" id="tax" label="Tax" variant="standard" fullWidth />
+            </Grid>
+
+            <Grid item xs={1.2}>
+              <TextField type="number" id="price" label="Price" variant="standard" fullWidth />
+            </Grid>
+
+            <Grid item container xs={1.2} alignItems={'center'} justifyContent={'center'}>
+              {/* Pytanie: Jak zrobić autoimport do ikon */}
+              <IconButton aria-label="delete">
```

Comment:
I think it should work out of the box, are you using Webstorm or VS Code?
<img width="538" alt="Screenshot 2023-08-02 at 09 15 35" src="https://github.com/broslukasz/react-upskill-basics/assets/84719303/cbbf1fa5-cff2-44b4-a674-24575f0047b2">

Reply:
vs code